projects/aws/upgrader/Makefile

```diff
@@ -0,0 +1,33 @@
+BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
+
+REPO=upgrader
+REPO_OWNER=aws
+
+GIT_TAG=v$(RELEASE_BRANCH)-$(shell yq e ".releases[] | select(.branch==\"${RELEASE_BRANCH}\").number" $(BASE_DIRECTORY)/EKSD_LATEST_RELEASES)
```

Comment:
I think im tired of seeing this in project makefiles... leave this for now, but I wil take an AI to look at moving this to common.mk somehow now that we have used it a few times.